build.gradle

```diff
@@ -117,7 +117,7 @@ subprojects {
                 )
 
                 option("NullAway:OnlyNullMarked", "true")
-                option("NullAway:CustomContractAnnotations", "io.micrometer.common.lang.internal.Contract")
+                option("NullAway:CustomContractAnnotations", "io.micrometer.common.lang.internal.Contract,org.assertj.core.internal.annotation.Contract")
```

Comment:
Initially, I thought the custom `@Contract` in AssertJ should be an internal detail, hence the `org.assertj.core.internal` package.

However, user projects would need to configure it like here,  and it might feel weird to configure something from an internal package.

I'll discuss this point internally.

Reply:
It should be but it seems right now NullAway has no discovery mechanism so we need to do this for every `@Contract` annotation.

Even if it is weird to do this, I would consider keeping it internal in case NullAway will have a discovery mechanism or JSpecify adds it making this option obsolete.

Reply:
We'll keep it internal in version 3 and revise it in version 4, based on user feedback and how JSpecify and NullAway will evolve.